assets/js/amp-story-editor-blocks.js

```diff
@@ -447,9 +472,27 @@ var ampStoryEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 			}
 
 			if ( 'amp/amp-story-grid-layer-thirds' !== parentBlock.name ) {
```

Comment:
@jacobschweitzer Thanks for all the changes! Otherwise looks good but the control probably should be out of the `if ( 'amp/amp-story-grid-layer-thirds' !== parentBlock.name ) {` condition -- otherwise it won't show up if it's an image within Thirds layer. Or was this something you purposefully added?

Reply:
@miina Good catch, I've moved it to `getAnimationControls)` which I've renamed to `getAmpStorySettings`. Is that ok ?